From 812b8bfd129a5aec65e4d14acdc746a8ff117f98 Mon Sep 17 00:00:00 2001
From: Ian Pozella <Ian.Pozella@imgtec.com>
Date: Thu, 16 Feb 2017 10:42:22 +0000
Subject: [PATCH 03/28] MIPS: DTS: add img directory to Makefile

An img directory exists for the Pistchio Soc but the directory
itself isn't in the dts Makefile meaning the dtbs never get built.

Signed-off-by: Ian Pozella <Ian.Pozella@imgtec.com>
---
 arch/mips/boot/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/boot/dts/Makefile b/arch/mips/boot/dts/Makefile
index fc7a0a9..b9db492 100644
--- a/arch/mips/boot/dts/Makefile
+++ b/arch/mips/boot/dts/Makefile
@@ -1,5 +1,6 @@
 dts-dirs	+= brcm
 dts-dirs	+= cavium-octeon
+dts-dirs	+= img
 dts-dirs	+= ingenic
 dts-dirs	+= lantiq
 dts-dirs	+= mti
-- 
2.7.4

